llvm/docs/LangRef.rst

```diff
@@ -26865,7 +26865,8 @@ Syntax:
 Arguments:
 """"""""""
 
-The first argument is a pointer. The second argument is an integer.
+The first argument is a pointer or vector of pointers. The second argument is
+an integer or vector of integers.
```

Comment:
nit: the working below still refers to a single pointer, maybe adjust to something like `pointer(s)`?